### ISSUE TITLE
cswin_large_224 pretrained 22k model

### ISSUE THREAD

**Issue by kaierlong** [BR-IDL/PaddleViT#212]
where can I get the cswin_large_224 pretrained 22k model and the 22kto1k labels mapping file.
I want to fine-tune the cswin_large_224 model with imagenet-1k dataset myself.

**Comment by xperzy** [comment]
Hello, the ported weight file is update [here](https://github.com/BR-IDL/PaddleViT/tree/develop/image_classification/CSwin#models-zoo).

**Comment by kaierlong** [comment]
@xperzy 
thanks for adding the pretrained 22k model. And where can I get the mapping file which map 22k label id to 1k label id. If it doesn't exist, how can I generate it?
Thanks!

**Comment by xperzy** [comment]
Hello, I don't think you need to have the mapping file for fine-tuning on imagenet 1k.   The last linear layer of the model (the 'head' layer) is different when you load the pretrained 22k and train on 1k model, this linear layer is considered to be trained from scratch for 1k classes.

**Comment by kaierlong** [comment]
@xperzy OK, I get it. Thanks again!
